VTEX - Catalog API Seller Portal.json

```
@@ -0,0 +1,4595 @@
+                                                                }
+                                                            }
+                                                        ],
+                                                        "items": {
+                                                            "type": "object",
+                                                            "title": "",
+                                                            "example": {
+                                                                "value": {
+                                                                    "id": "2",
+                                                                    "name": "Perfumes",
+                                                                    "isActive": false
+                                                                }
+                                                            },
+                                                            "required": [
+                                                                "value"
+                                                            ],
+                                                            "properties": {
+                                                                "value": {
+                                                                    "type": "object",
+                                                                    "title": "value",
+                                                                    "description": "Object with values of a children category.",
+                                                                    "example": {
+                                                                        "id": "2",
+                                                                        "name": "Perfumes",
+                                                                        "isActive": false
+                                                                    },
+                                                                    "required": [
+                                                                        "id",
+                                                                        "name",
+                                                                        "isActive"
+                                                                    ],
+                                                                    "properties": {
+                                                                        "id": {
+                                                                            "type": "string",
+                                                                            "title": "id",
+                                                                            "description": "Children category unique identifier number."

```

```suggestion
                "description": " >📘 This API is part of the [Seller Portal Catalog](https://help.vtex.com/en/tutorial/how-the-seller-portal-catalog-works--7pMB6YOt6YQDQQbzFB4Pxp). This functionality is in the Beta stage and can be discontinued at any moment at VTEX's discretion. VTEX will not be responsible for any instabilities caused by its use or discontinuity. If you have any questions, please contact [our Support Center](https://support.vtex.com/hc/en-us/requests). \r\n\r\n Creates a new category.\r\n\r\n## Request body example\r\n\r\n```json\r\n{\r\n  \"parentId\": \"567\",\r\n  \"Name\": \"Beauty\"\r\n}\r\n```\r\n\r\n## Response body example\r\n\r\n```json\r\n{\r\n  \"value\": {\r\n    \"id\": \"1\",\r\n    \"name\": \"Beauty\",\r\n    \"isActive\": false\r\n  },\r\n  \"children\": [\r\n    {\r\n      \"value\": {\r\n        \"id\": \"2\",\r\n        \"name\": \"Perfumes\",\r\n        \"isActive\": false\r\n      }\r\n    }\r\n  ]\r\n}\r\n```",
```